spec/tbd_tests_spec.rb

```diff
@@ -6845,7 +6853,7 @@
 
     # Same exercise, yet provide clerestory with Frame & Divider.
     fd2    = OpenStudio::Model::WindowPropertyFrameAndDivider.new(model_FD)
-    width2 = 0.03
+    width2 = 0.06
```

Comment:
This adjustment is now required for OpenStudio SDK versions v3.2.1 through v3.3.0, as described [here](https://github.com/rd2/osut/pull/17#discussion_r1554989291). Previously, TBD only relied on _OSut_'s (older) `overlaps?` method (returning true or false) to detect whether 2 polygons intersected (e.g. when dealing with Frame & Divider widths). _OSut_'s `overlaps?` will soon depend on a related `overlap` method (which instead returns the actual intersecting polygon, relying on [OpenStudio.intersect](https://github.com/NREL/OpenStudio/blob/2d5ba97179292d7c836444566d0ccb34a2c18c5a/src/utilities/geometry/Intersection.cpp#L806) function). As discussed in the quoted _OSut_ PR comment, v3.2.1 and v3.3.0 `OpenStudio.intersect` versions do not catch narrow intersections (like narrow window frames). Version v3.0.0, as well as any version > 3.3.0, work just fine. So simply relaxing the test here.